docs/en/spiffs.md

```diff
@@ -0,0 +1,118 @@
+# SPIFFS File System
+
+The NodeMCU project uses the SPIFFS filesystem to store files in the flash chip. The technical details about how this is configured can be found below, along with vairous build time options.
+
+# spiffsimg - Manipulate SPI Flash File System disk images
+
+Ever wished you could prepare a SPIFFS image offline and flash the whole
+thing onto your microprocessor's storage instead of painstakingly upload
+file-by-file through your app on the micro? With spiffsimg you can!
+
+NodeMCU uses a SPIFFS filesystem that knows how big it is -- i.e. when you build a file system
+image, it must fit into the flash chip, and it cannot be expanded once flashed.
+It is important to give the `spiffimg` tool the correct size. You can provide either the `-c` option or both the `-U` and `-S` options.
+
+### Syntax 
+
+`spiffsimg -f <filename> 
```

Comment:
I think you want to use 3 back ticks instead of one - otherwise it's all be on a single line.